src/ocm/components/HostsClusterDetailTab/HostsClusterDetailTabContent.tsx

```diff
@@ -101,7 +104,8 @@ export const HostsClusterDetailTabContent = ({
             ocmCluster,
             pullSecret,
             normalizedVersion,
-            extraInfo,
+            cpuArchitectures,
+            canSelectCpuArch,
           );
           setDay2Cluster(
```

Comment:
The call to `setDay2Cluster` still passes the `extraInfo` parameter.
In fact,  there are still a few references to it in some places, for example `HostsClusterDetailTabProps`, the type `OcmClusterExtraInfo`, etc.

I suggest you look for "extraInfo" to remove all remaining references to it.